src/Conversion/ONNXToKrnl/ConvertONNXToKrnl.cpp

```diff
@@ -55,73 +57,99 @@ void populateONNXToKrnlConversionPattern(RewritePatternSet &patterns,
 
   // Frontend operation lowering.
   // ControlFlow
-  populateLoweringONNXLoopOpPattern(patterns, typeConverter, ctx);
-  populateLoweringONNXScanOpPattern(patterns, typeConverter, ctx);
+  onnx_mlir::populateLoweringONNXLoopOpPattern(patterns, typeConverter, ctx);
```

Comment:
You already have `using namespace onnx_mlir;` at line 22, but do you need to prefix 1populateLoweringONNX.*Pattern` with `onnx_mlir::`?

Reply:
done